app/src/main/java/org/sopt/zooczoocbbangbbang/presentation/main/mypage/MyProfileEditCustomDialog.kt

```diff
@@ -0,0 +1,30 @@
+package org.sopt.zooczoocbbangbbang.presentation.main.mypage
+
+import android.app.Dialog
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.widget.Button
+import org.sopt.zooczoocbbangbbang.R
+
+class MyProfileEditCustomDialog(context: Context,
+    MyCustomDialogInterface: MyCustomDialogInterface)
+    : Dialog(context) {
```

Comment:
```suggestion
class MyProfileEditCustomDialog(
    context: Context,
    MyCustomDialogInterface: MyCustomDialogInterface
) : Dialog(context) {
```
다음과 같이 고치는 것이 보다 가독성이 좋고 컨밴션입니다 :)
https://kotlinlang.org/docs/coding-conventions.html#class-headers

또한 해당 Dialog 는 설계 구조상 interface를 구현한 구현채가 Activity가 된다면 사용 위치에서 각 매개변수에 this, this 가 들어가기 때문에 의도를 쉽게 파악하기 힘들다고 생각해

차라리 고차 함수를 이용하여 람다를 받을 수 있도록 설계하는 것은 어떨까?